src/components/Badge/tests/Badge.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Badge from '../Badge';
+
+describe('<DisplayText />', () => {
+
+    it('Verify the default badge.', () => {
```

Comment:
just a minor suggestion: the tests should be a complete sentence. example - it('should verify the default badge.')